odoo80/Dockerfile

```diff
@@ -1,51 +1,5 @@
-FROM vauxoo/docker-ubuntu-base
+FROM vauxoo/docker-ubuntu-base:latest
 MAINTAINER Tulio Ruiz <tulio@vauxoo.com>
 
-RUN wget -qO- https://deb.nodesource.com/setup | sudo bash - \
-    && apt-get upgrade -q \
-    && apt-get install --allow-unauthenticated -q \
-    antiword \
-    bash-completion \
-    libfreetype6-dev \
-    libgeoip-dev \
-    libjpeg-dev \
-    libldap2-dev \
-    libqrencode-dev \
-    libxml2-dev \
-    libxslt1-dev  \
-    libyaml-dev \
-    M2Crypto \
-    nodejs \
-    phantomjs \
-    poppler-utils \
-    python-imaging \
-    python-libxml2 \
-    python-lxml \
-    python-pandas \
-    python-qrcode \
-    ttf-dejavu \
-    xmlstarlet \
-    xsltproc \
-    zlib1g-dev \
-    && npm -g install npm@latest \
-    && npm install -g less \
-    && npm install -g less-plugin-clean-css \
-    && pip install --upgrade pyyaml pillow GeoIP SOAPpy pyopenssl suds xmltodict recaptcha-client egenix-mx-base git+https://github.com/Vauxoo/panama-dv.git \
-    && cd /tmp && git clone --depth=1 https://github.com/thewtex/sphinx-contrib.git \
-    && cd sphinx-contrib/youtube && python setup.py install \
-    && wget -q https://raw.githubusercontent.com/ruiztulio/gist-vauxoo/master/travis_run.py \
-    && python travis_run.py \
-    && wget -q https://raw.githubusercontent.com/Vauxoo/odoo-network/8.0/addons/network/scripts/odoo-server/05-install-dependencies-python-v80.sh \
-    && sh 05-install-dependencies-python-v80.sh \
-    && wget -q https://raw.githubusercontent.com/vauxoo/odoo/8.0/requirements.txt \
-    && pip install -r requirements.txt \
-    && wget -q http://geolite.maxmind.com/download/geoip/database/GeoLiteCity.dat.gz \
-    && gunzip GeoLiteCity.dat.gz && mkdir -p /usr/share/GeoIP && mv GeoLiteCity.dat /usr/share/GeoIP/GeoLiteCity.dat \
-    && wget -O wkhtmltox-0.12.1_linux-trusty-amd64.deb http://download.gna.org/wkhtmltopdf/0.12/0.12.1/wkhtmltox-0.12.1_linux-trusty-amd64.deb \
-    && dpkg -i wkhtmltox-0.12.1_linux-trusty-amd64.deb \
```

Comment:
I [changed this](https://github.com/Vauxoo/docker-odoo-image/pull/104/commits/7b4fdbcfb35a9578b888154c04acadfbbd8bb007#diff-47daa48a95db7533add915692de62084R65) to a more elegant helper function.